web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -270,20 +246,28 @@ BmcMangaList.prototype.filter = function(filterStr) {
     }
 };
 
+function showDeleteButton() {
+    displayButton(document.getElementById('delete-but'));
+    document.getElementById('register-but').style.display = '';
+}
+
 function showRegisterButton() {
+    displayButton(document.getElementById('register-but'));
+    document.getElementById('delete-but').style.display = '';
+    document.getElementById('side-adder').disabled = false;
+}
+
+function displayButton(btn) {
```

Comment:
We should rationalize this function into a Class handling the states of the SideBar.
In thie specific case (to summarize), this function shows the button if and only if the panel is hidden.

Reply:
Will be handled through #53